packages/grafana-e2e/src/flows/configurePanel.ts

```diff
@@ -90,7 +90,13 @@ export const configurePanel = (config: PartialAddPanelConfig | PartialEditPanelC
       e2e.components.Panels.Panel.title(panelTitle).click();
       e2e.components.Panels.Panel.headerItems('Edit').click();
     } else {
-      e2e.components.PageToolbar.item('Add panel').click();
+      try {
+        // Depending on the screen size, the "Add panel" button might be hidden
+        e2e.components.PageToolbar.item('Show more items').click();
+        e2e.components.PageToolbar.item('Add panel').last().click();
+      } catch (e) {
+        e2e.components.PageToolbar.item('Add panel').click();
+      }
```

Comment:
i think this would make more sense the other way around? e.g. try to click `Add panel`, if that errors then try to click `Show more items` -> `Add panel`